test/Poseidon/GoldenTestsRunCommands.hs

```diff
@@ -110,12 +114,22 @@ testPipelineInit testDir checkFilePath testPacsDir = do
         , _initPacPath   = testDir </> "Wang"
         , _initPacName   = "Wang"
     }
-    runAndChecksumFiles checkFilePath testDir (runInit initOpts2) "init" [
+    let action2 = runInit initOpts2 >> patchLastModified testDir ("Wang" </> "POSEIDON.yml")
+    runAndChecksumFiles checkFilePath testDir action2 "init" [
           "Wang" </> "POSEIDON.yml"
         , "Wang" </> "Wang.janno"
         , "Wang" </> "Wang_2020.bed"
         ]
 
+patchLastModified :: FilePath -> FilePath -> IO ()
+patchLastModified testDir yamlFile = do
+    lines_ <- T.lines <$> T.readFile (testDir </> yamlFile)
```

Comment:
Isn't there an issue with laziness here? Reading and overwriting causes issues sometimes, when done with readFile and writeFile, right? Is Text.readFile strict?

Reply:
Yes it is in Data.Text.IO. I had used the lazy String version before from Prelude and promptly stumbled into problems. That's why I used the one from Data.Text.IO

Reply:
Good to know. I solved this issue recently by deleting the file before rewriting it. Probably an explicitly strict solution is less of a hack...

https://github.com/poseidon-framework/poseidon-hs/blob/6652c6b2f1b44f0c9a09f084e66d39495454e4c5/src/Poseidon/CLI/Update.hs#L83-L86